.github/workflows/test-golang.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: Run Golang tests
+
+on:
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  go_mod_verify:
```

Comment:
update mergify rules for new jobs?

Reply:
yes, but only after this is merged by Mergify